stroom-kafka-client-impl_0_10_0_1/src/test/java/stroom/connectors/kafka/TestStroomKafkaProducer.java

```diff
@@ -0,0 +1,64 @@
+package stroom.connectors.kafka;
+
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+import stroom.connectors.ConnectorProperties;
+import stroom.connectors.ConnectorPropertiesEmptyImpl;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.function.Consumer;
+
```

Comment:
I moved this in with the kafka provider library, to prevent any pollution of the kafka client libraries in our main app (even via test deps)